<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogprogram_detection</class>
 <widget class="QDialog" name="dialogprogram_detection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget_JobList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Get_JobList">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Job List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add_Job">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Job</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Run_Job">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Job</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Go_PickPosition">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Pick Position</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_Z">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Z distance :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add_Go_PickPosition">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Go Pick Position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Go_AnglePosition">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Angle Position</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Edit_Angle">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dest Angle :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
